ipn/ipnlocal/cert.go

```diff
@@ -53,8 +53,8 @@ var (
 	// populate the on-disk cache and the rest should use that.
 	acmeMu sync.Mutex
 
-	renewMu        sync.Mutex // lock order: don't hold acmeMu and renewMu at the same time
-	lastRenewCheck = map[string]time.Time{}
+	renewMu     sync.Mutex // lock order: don't lock acmeMu while renewMu is locked
```

Comment:
```// lock order: acmeMu before renewMu```

Reply:
Convention is to say the order in which they should be acquired if both need to be acquired, if any one of them needs to be acquired independently, that is allowed.